docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,287 @@
+and uses an EC2 instance to serve the Teleport Database Access service.
+The level of access provided may not suit your needs, or may not fit your
+organization's access conventions. You should adjust the AWS IAM permissions to fit your needs.
+</Admonition>
+
+## Step 1/4. Create IAM roles for DynamoDB access
+
+The setup described in this guide requires two IAM roles: one associated with the EC2 instance running the Teleport Database Access service and another that can be assumed by the EC2 instance role and grants access to DynamoDB services.
+
+### EC2 instance role
+
+Visit the [IAM > Roles page](https://console.aws.amazon.com/iamv2/home#/roles) of
+the AWS Console, then press "Create Role". Under **Trusted entity type** select
+"AWS service". Under **Use case** select "EC2", then click **Next**.
+
+![Create Role to Identify EC2 Instance](../../../img/database-access/guides/dynamodb-create-ec2-role.png)
+
+On the "Add Permissions" page, you can simply click **Next** since this role does not require any permissions. In this guide, we will use the example name `TeleportDatabaseService` for this role. Once you have chosen a name, click **Create Role** to complete the process.
+
+### DynamoDB access role
+
+Navigate back to the Roles page and create a new role. Select the "AWS account"
+option, which creates a default trust policy to allow other entities in this
+account to assume this role:
+
+![Create Role Step 1](../../../img/database-access/guides/dynamodb-create-role-1.png)
+
+Click **Next**. Find the AWS-managed policy `AmazonDynamoDBFullAccess` and then select the policy:
+
+![Create Role Step 2](../../../img/database-access/guides/dynamodb-create-role-2.png)
+
+<Admonition type="note" title="Apply least-privilege permissions">
+The `AmazonDynamoDBFullAccess` policy may grant more permissions than desired.
+If you want to use a different IAM policy to reduce permissions, refer to
+[Managing access permissions to your Amazon DynamoDB 
+Resources](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/access-control-overview.html)
+for more information.
+</Admonition>
+
+Click **Next**. On the next page, enter a role name. In this guide we'll use
+the example name `ExampleTeleportDynamoDBRole` for this role.
+
+Under "Select trusted entities", update the JSON to allow the `TeleportDatabaseService`
+role to assume this role:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::(=aws.aws_access_key=):role/TeleportDatabaseService"
+                ]
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+    ]
+}
+```
+
+Finally, click **Create Role**.
+
+## Step 2/4. Configure the Teleport IAM role mapping
+
+The next step is to give your Teleport users permissions to assume IAM roles in
```

Comment:
Does this sentence mean the same thing as "...give your Teleport users permissions to assume IAM roles when accessing AWS resources registered with Teleport", or something else?

Reply:
Clarified in 6aa275864c7ce42725be77d3fef8699fd3b90ec0